raiden/tests/unit/test_channelstate.py

```diff
@@ -225,9 +226,11 @@ def test_new_end_state():
 def test_endstate_update_contract_balance():
     """The balance must be monotonic."""
     balance1 = 101
+    claim = deepcopy(EMPTY_CLAIM)
+    claim.total_amount = balance1
```

Comment:
FYI: This can also be done with https://docs.python.org/3/library/dataclasses.html#dataclasses.replace
```
claim = dataclasses.replace(claim, total_amount=balance1)
```

Reply:
this was refactored by `make_claim()`